src/api/java/com/minecolonies/api/entity/MinecoloniesMinecart.java

```diff
@@ -0,0 +1,291 @@
+package com.minecolonies.api.entity;
+
+import com.google.common.collect.Maps;
+import com.mojang.datafixers.util.Pair;
+import net.minecraft.block.AbstractRailBlock;
+import net.minecraft.block.BlockState;
+import net.minecraft.block.PoweredRailBlock;
+import net.minecraft.entity.Entity;
+import net.minecraft.entity.EntityType;
+import net.minecraft.entity.item.minecart.AbstractMinecartEntity;
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.network.IPacket;
+import net.minecraft.state.properties.RailShape;
+import net.minecraft.util.Direction;
+import net.minecraft.util.Hand;
+import net.minecraft.util.Util;
+import net.minecraft.util.math.*;
+import net.minecraft.world.World;
+import net.minecraftforge.fml.network.NetworkHooks;
+import org.jetbrains.annotations.NotNull;
+
+import javax.annotation.Nullable;
+import java.util.Map;
+
+/**
+ * Special minecolonies minecart that doesn't collide.
+ */
+public class MinecoloniesMinecart extends AbstractMinecartEntity
+{
+    /**
```

Comment:
Do we want this to ignore powered rails and such? should overwrite 
    /**
     * Return false if this cart should not call onMinecartPass() and should ignore Powered Rails.
     * @return True if this cart should call onMinecartPass().
     */
    default boolean shouldDoRailFunctions() {
        return true;
    }
then

Reply:
Wanted to keep it as is for now. If we notice problems with them we can still turn them off.